src/ast/mod.rs

```diff
@@ -251,7 +251,7 @@ pub enum Expr {
     },
     MapAccess {
```

Comment:
I wonder if we can improve the docstrings here while we are modifying this file?